ydb/library/yql/dq/opt/dq_opt_conflict_rules_collector.cpp

```diff
@@ -0,0 +1,86 @@
+#include "dq_opt_conflict_rules_collector.h"
+#include <util/generic/hash_set.h>
+
+namespace NYql::NDq {
+
+/* To make ASSOC, RASSCOM, LASSCOM tables simplier */
+EJoinKind GetEquivalentByAlgebraicPropertiesJoin(EJoinKind joinKind) {
```

Comment:
The name is broken English

Better name: GetEquivalentJoinByAlgebraicProperties